src/database/index.tsx

```diff
@@ -162,9 +183,11 @@ export const getFacilities = async (): Promise<Array<Facility> | null> => {
       .collection(facilitiesCollectionId)
       .get();
 
-    const facilities = facilitiesResults.docs.map(
-      (doc) => doc.data() as Facility,
-    );
+    const facilities = facilitiesResults.docs.map((doc) => {
+      const facility = doc.data() as Facility;
+      facility.id = doc.id;
+      return facility;
+    });
```

Comment:
Adding the CloudStore docRef `id` onto each facility here so that we know which to persist when an individual Facility is selected and needs to be updated